packages/python-pypdf/PKGBUILD

```diff
@@ -0,0 +1,28 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+# This file is based on the AUR package by mark.blakeney at bullet-systems dot net
+
+_name=pypdf
```

Comment:
```suggestion
_pkgname=pypdf
```

Convention is to use `_pkgname` instead of `_name`.

Reply:
Despite `${pkgname#python-}` is empty, the variable remains empty when `makepkg` is run.

Reply:
Applied.